.github/workflows/conda-cpp-build.yaml

```diff
@@ -72,6 +74,7 @@ jobs:
 
           echo "MATRIX=${MATRIX}" | tee --append "${GITHUB_OUTPUT}"
   build:
+    name:  ${{ matrix.CUDA_VER }}, ${{ matrix.PY_VER }}, ${{ matrix.ARCH }}, ${{ matrix.LINUX_VER }}
```

Comment:
was this change intended to be included in this PR? it's not mentioned in the PR body. I know others may have thoughts on what type of naming convention we use, so it might be best to postpone these changes for now.

Reply:
should we at least have something there until there is a consensus on the naming convention we should have? I'm trying to clear out an inconsistency where the wheels build job is named but the conda job isn't.

I can include the change in the PR description

Reply:
that's fine